README.md

```diff
@@ -35,6 +35,11 @@ dependencies {
 }
 ```
 
+#### ProGuard
```

Comment:
This is not the right spot in the README for this. Just remove this section for now.

Reply:
Sure, we could add in Android-specific README.md in the future.